webhook/handlers/patient_first_workflow.js

```diff
@@ -0,0 +1,71 @@
+const db = require('../../models')
+
+class PatientFirstController {
+
+  async bookConsulation(agent, id){
+    let fulfillmentText = "";
+    let user = await db.userWA.findOne({where: {wa_phone_number: id}});
+    let patients = await db.patientWA.findAll({where: { user_id : user.id }});
+        if(patients.length == 0){
+            fulfillmentText += ("We don't have any patients registered with us. What would you like to do?\
+            \n 1. Register a patient?\
+            \n 2. Connect with the Hospital immediately?\
+            \n 3. More info?");
+        }
+        else
+        {
+            patients.forEach((patient, i) => {
+                fulfillmentText += ("\nWe have following patient profiles with us: \n" + i + 1  + ": " +
```

Comment:
This line is being added into fulfillment text multiple times.
Output:
```
We have the following patient profiles with us: 
1: Perry Platypus
We have the following patient profiles with us: 
2: Phineas Flynn
```

Reply:
Use this instead-
https://github.com/kj7kunal/E-Med/blob/develop-patient-first-workflow/webhook/handlers/user_registration_workflow.js#L193-L197

Reply:
`+ i + 1 +` needs to be converted to `(i+1)` OR `+ (i+1).toString()`.